packages/next/taskfile.js

```diff
@@ -611,6 +611,12 @@ export async function ncc_path_browserify(task, opts) {
       target: 'es5',
     })
     .target('compiled/path-browserify')
+
+  const filePath = join(__dirname, 'compiled/path-browserify/index.js')
+  const content = fs.readFileSync(filePath, 'utf8')
+
+  // Remove process usage from path-browserify polyfill for edge-runtime
+  await fs.writeFile(filePath, content.replace(/process\.cwd\(\)/, '""'))
```

Comment:
```suggestion
  await fs.writeFile(filePath, content.replace(/process\.cwd\(\)/g, '""'))
```
Can there be multiple occurrences of `process.cwd()` that need replacing?

Reply:
there's only 1, but the global flag look safer we can patch with it